.husky/pre-commit.js

```diff
@@ -180,14 +186,14 @@ function getChangedFileExtensions(_folder) {
 }
 
 /**
- * Execute linting command
+ * Execute commands based on file extensions
  * @param {FOLDER, FILE_EXTENSION[]}
  */
-function executeLinting(_folder, _changedFileExtensions) {
+function executeCommands(_folder, _changedFileExtensions, _command) {
```

Comment:
```suggestion
function executeCommand(_folder, _changedFileExtensions, _command) {
```
Suggest for consistency with param `_command` :)